nengo_dl/tests/test_benchmarks.py

```diff
@@ -154,6 +155,34 @@ def test_cli():
     sys.argv = old_argv
 
 
+@pytest.mark.training
+@pytest.mark.parametrize("native_nengo", (True, False))
+def test_lmu(Simulator, native_nengo, pytestconfig):
+    n_steps = 1000
+    net = benchmarks.lmu(
+        n_steps, 1, native_nengo=native_nengo, dtype=pytestconfig.getoption("--dtype")
+    )
+
+    # benchmarks.run_profile(
```

Comment:
Should this commented out code be removed?

Reply:
I left it in because there's a story to look into optimizing LMU performance (which would use this profiling snippet here). Wouldn't be hard to recreate though, so we could also just remove it 🤷‍♂.

Reply:
Fine to leave it in I think, maybe add a `todo` comment at the top? I can do that in the merge if you like.

Reply:
Yeah that sounds good